### DISCUSSION TITLE
My Nano Els setup with a hollow frame mini lathe

### DISCUSSION THREAD

**Discussion by EdFleta** [kachurovskiy/nanoels#72] [Show and tell]
Hi everybody,

I thought it could be of some interest to see my setup. (still in progress)
I bought this mini lathe in July from Vevor and although it resembles many of the WM210V I have seen before, it has a hollow frame 
![Eddie_setup](https://user-images.githubusercontent.com/113701009/190862508-5759b970-dd57-49f5-b12a-c66dddd31dcf.jpg)
. This particular feature  allowed me to tuck the encoder in the cavity that houses the speed indicator and the knob.

The black cover is a 3D printed part as well as the pullies (the encoder one reinforced with an aluminium center part) in a 100/80 ratio to allow the cover to close.

The spindle pully is slide over the tachometer ring and does not seem to interfere with the magnets.

Thanks to Maxim recommendations, after modifying the code to 750 steps instead of 600 for the encoder and reverse the A/B pins , everything works fine.

I have all the parts in STL and freecad formats if needed.

best regards,

eddie

**Comment by kachurovskiy** [comment]
Awesome work, hope it doesn't disappoint in the long run!

**Reply by EdFleta** [reply]
why shouldt it? is there any technical issue I should be aware of re this setup?

**Reply by kachurovskiy** [reply]
Depending on the plastic you used for printing, the motor mount could start to bow if you run the motor for a long time and it heats up. Assuming this is a hobby setup, I wouldn't worry about it too much.

**Reply by EdFleta** [reply]
You are right: I have used PLA for this test part. I will switch to PETG when everything is finalised. I have also ordered metal pullies and I am machining the motor mount in aluminium.
Then I expect the whole thing to be strurdy enough. 

I will get familiar with it until I am ready to upgrade it with a two axis system as the one you demonstrated.

**Comment by gsherwin74** [comment]
How are you liking the lathe.  I just got one and am now thinking of adding an H2 to it.

**Reply by EdFleta** [reply]
Hi, As said,the version I have is slightly different from those I've seen from other suppliers. However they all have one thing in common that is the lack of rigidity. It is definitely like 3d printers: you probably have to spend a great deal of time using the machine to improve it before you can use it for real projects. At least it is my experience.
I am also implementing H2 on my lathe.

**Reply by Johannmupa** [reply]
Lack of Rigidity
I was used to work with a CNC lathe in 210 by 400 size for the parts but the machine weight was about 5 tons. With some operations you can get even this heavy machine to chatter. The trick is to find speed and feed rates that avoid chatter. An ELS is a great advantage for this.

**Comment by fraidd** [comment]
bonjour
j'adore vos poulie en jaune ou avez vous trouvé le stl?
merci a vous
fraidd
